### PR TITLE
Add capacity to preview image of thread

### PR DESCRIPTION
**Pull request by RoiSoleil**

### CODE REVIEW

File: track.c [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *    Copyright 2000, Jeroen Vreeken
  *    This program is published under the GNU Public license
  */
-
+#include <linux/videodev2.h>
```
Comment (tosiara) [inline track.c:8]: Have you added this to workaround compilation error?

**Comment by tosiara** [comment]
Can you explain this pull? Does it add some new functionality or resolves an issue?

File: track.c (hunk 1 above)
Comment (RoiSoleil) [inline track.c:8]: Yes i did it to be able to compile on Ubuntu

Comment (tosiara) [inline track.c:8]: So this is a known issue, however it is not motion's fault, but rather OS (only some affected). I don't think it is worth changing the code just to workaround that issue. Check this thread: https://github.com/Mr-Dave/motion/pull/71

Comment (RoiSoleil) [inline track.c:8]: OK for me

**Pull request closed without merging**